public-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -19,10 +23,22 @@
 import org.springframework.data.domain.Sort.Direction;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Service;
+import org.pmiops.workbench.cdr.ConceptMapper;
+import org.pmiops.workbench.model.SearchConceptsRequest;
+import org.pmiops.workbench.model.ConceptListResponse;
+import org.pmiops.workbench.model.Domain;
+import org.pmiops.workbench.model.StandardConceptFilter;
 
 @Service
 public class ConceptService {
 
+    @PersistenceContext(unitName = "cdr")
+    private EntityManager entityManager;
+    @Autowired
+    private ConceptDao conceptDao;
+    @Autowired
+    private ConceptMapper conceptMapper;
+
     public static enum SearchType {
```

Comment:
Modifier `static` is redundant for inner enums. Can be removed.
```suggestion
    public enum SearchType {
```

Reply:
Changed this.